src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));
```

Comment:
make global Random and ColorSupplier and use them here

Reply:
Which class i have to make global Random and ColorSupplier?

Reply:
> make global Random and ColorSupplier and use them here

Which class i have to make global Random and ColorSupplier?